pyScienceMode2/motomed_interface.py

```diff
@@ -5,7 +5,7 @@
 
 from pyScienceMode2.acks import *
```

Comment:
Avoid general import and import actual Class

Reply:
Done.